internal/core/software_renderer.rs

```diff
@@ -976,7 +980,7 @@ impl<'a, T: ProcessScene> SceneBuilder<'a, T> {
                                         buf_size.height / img_src_size.height as f32,
                                     )
                                     .cast(),
-                                colorize,
+                                colorize: self.alpha_color(colorize),
```

Comment:
If colorize is fully transparent, this will have no effect.

`colorize` is there to apply a color effect on the image. The sofwtare renderer check if colorize is transparent and then it doesn't colorize it (instead of drawing the image fully transarent) 

So we need another field to store the alpha in that case.

Reply:
Ok I see. I added a new alpha field.